src/quo2/components/buttons/predictive_keyboard/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.buttons.predictive-keyboard.view
+  (:require [react-native.core :as rn]
+            [quo2.components.buttons.predictive-keyboard.style :as style]
+            [quo2.components.info.info-message :as info-message]
+            [react-native.linear-gradient :as linear-gradient]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.buttons.button :as button]))
+
+(def gradients
```

Comment:
Following the spirit of exposing only `view`, we can add `^:private` to this var's metadata.